website/content/docs/connect/ca/index.mdx

```diff
@@ -4,6 +4,23 @@ page_title: Connect - Certificate Management
 description: An overview of the Connect Certificate Authority mechanisms.
 ---
 
+### CA and Certificate relationship
```

Comment:
This section should be moved after the main heading/intro paragraph.  So after # Connect Certificate Management.

Reply:
done